packages/backend/src/graphql/mutations/update-flow-owner.ts

```diff
@@ -0,0 +1,48 @@
+import Flow from '@/models/flow'
+import Step from '@/models/step'
+import Context from '@/types/express/context'
+
+type Params = {
+  input: {
+    id: string
+  }
+}
+
+const updateFlowOwner = async (
```

Comment:
this query will allow any logged in user to take over a pipe, since there's no validation that a pipe transfer to the logged in user exists for this pipe.

potential ways to solve this:
* Query `flow_transfers` table validate (duh)
* Make `id` a flow_transfer id and validate from there.
  * You could also add a `HasMany` relation from `User` to `FlowTransfer.new_owner_id` . (bonus: this will enable you to directly query from `context.currentUser` inside the  `getPendingFlowTransfers` query)